src/Classes/Daemons/MyRadio_TrackAndTraceDaemon.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace MyRadio\Daemons;
+
+use MyRadio\Config;
+use MyRadio\ServiceAPI\MyRadio_Timeslot;
+use MyRadio\ServiceAPI\MyRadio_Season;
+use MyRadio\MyRadio\CoreUtils;
+use MyRadio\MyRadioEmail;
+use MyRadio\ServiceAPI\MyRadio_List;
+
+class MyRadio_TrackAndTraceDaemon extends \MyRadio\MyRadio\MyRadio_Daemon
+{
+    public static function isEnabled()
+    {
+        return Config::$d_TrackAndTrace_enabled;
+    }
+
+    public static function run()
+    {
+        $weekkey = __CLASS__ . '_last_run_weekly';
+        if (self::getVal($weekkey) > time() - 604700) {
+            return;
+        }
+
+        self::generateTrackAndTraceReport();
+
+        // Done
+        self::setVal($weekkey, time());
```

Comment:
If generateTrackAndTraceReport yeets an exception, last_run won't be updated and myradio_daemon will loop infinitely, won't it? Is this intended behaviour?

Reply:
🤔

Reply:
i mean, lol exceptions